cookbook/deployment/customizing_resources.py

```diff
@@ -25,20 +25,22 @@
 # #. ``mem``
 # #. ``gpu``
 #
-# To ensure regular tasks that do not require GPUs are not scheduled on GPU nodes, a separate node group for GPU nodes can be configured with `taints <https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/>`_.
+# To ensure regular tasks that don't require GPUs are not scheduled on GPU nodes, a separate node group for GPU nodes can be configured with `taints <https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/>`_.
 #
-# To ensure tasks that do require GPUs get the needed tolerations on their pods, set up flytepropeller using the following `configuration <https://github.com/flyteorg/flytepropeller/blob/v0.10.5/config.yaml#L51,L56>`_. Make sure that this toleration config matches the taint config that you have configured to protect your gpu providing nodes from having to deal with regular non-gpu workloads (pods).
+# To ensure tasks that require GPUs get the needed tolerations on their pods, set up FlytePropeller using the following `configuration <https://github.com/flyteorg/flytepropeller/blob/v0.10.5/config.yaml#L51,L56>`_. Ensure that this toleration config matches the taint config you have configured to protect your gpu providing nodes from dealing with regular non-gpu workloads (pods).
 #
 # The ``storage`` resources option is not yet supported, but coming soon.
```

Comment:
Can you remove this line cause we have ``storage``: https://docs.flyte.org/projects/flytekit/en/latest/generated/flytekit.Resources.html#flytekit-resources